app/scripts/BarTrack.js

```diff
@@ -298,16 +307,23 @@ class BarTrack extends HorizontalLine1DPixiTrack {
     // but rather its parent's
     super.draw();
 
-    if (this.options.zeroLineVisible) this.drawZeroLine();
-    else this.zeroLine.clear();
+    if (this.zeroLine?._graphics && this.options.zeroLineVisible) {
```

Comment:
Comment about why the first portion of the if is necessary.

Reply:
Added comment to explain why this test is used.